lib/pxcontext.js

```diff
@@ -32,14 +32,20 @@ class PxContext {
             const tokenObject = this.getTokenObject(mobileHeader);
             this.cookies[tokenObject.key] = tokenObject.value;
         } else {
-            this.requestCookieNames = Object.keys(req.cookies);
-            Object.keys(req.cookies).forEach(key => {
+            let cookies;
```

Comment:
Perfectly fine that way.
But if I can suggest:

```
let cookies = req.cookies;
if (config.CUSTOM_COOKIE_HEADER && req.headers[config.CUSTOM_COOKIE_HEADER]) {
     cookies = pxUtil.parseCookieHeader(req.headers[config.CUSTOM_COOKIE_HEADER]);
}
```
You can keep it like that if you like